Riot/Categories/UIView+Toast.swift

```diff
@@ -0,0 +1,310 @@
+        view.backgroundColor = .clear
+        return view
+    }()
+    
+    private lazy var messageLabel: UILabel = {
+        let label = UILabel()
+        label.font = .systemFont(ofSize: 15)
+        label.backgroundColor = .clear
+        label.numberOfLines = 0
+        label.textAlignment = .left
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+    
+    private lazy var stackView: UIStackView = {
+        let result = UIStackView()
+        result.axis = .horizontal
+        result.distribution = .fill
+        result.alignment = .center
+        result.spacing = 8.0
+        result.backgroundColor = .clear
+        
+        addSubview(result)
+        result.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            result.leadingAnchor.constraint(equalTo: leadingAnchor, constant: Constants.padding.left),
+            result.topAnchor.constraint(equalTo: topAnchor, constant: Constants.padding.top),
+            result.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -Constants.padding.right),
+            result.bottomAnchor.constraint(equalTo: bottomAnchor, constant: -Constants.padding.bottom)
+        ])
+        
+        return result
+    }()
+    
+    init(withMessage message: String?,
+         image: UIImage? = nil) {
+        super.init(frame: .zero)
+        
+        if let image = image {
+            imageView.image = image
+            NSLayoutConstraint.activate([
+                imageView.widthAnchor.constraint(equalToConstant: image.size.width),
+                imageView.heightAnchor.constraint(equalToConstant: image.size.height)
+            ])
+            stackView.addArrangedSubview(imageView)
+        }
+        
+        messageLabel.text = message
+        stackView.addArrangedSubview(messageLabel)
+        
+        stackView.layoutIfNeeded()
+        layer.cornerRadius = Constants.cornerRadius
+        layer.masksToBounds = true
+        registerThemeServiceDidChangeThemeNotification()
+        themeDidChange()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private func registerThemeServiceDidChangeThemeNotification() {
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(themeDidChange),
+                                               name: .themeServiceDidChangeTheme,
+                                               object: nil)
+    }
+    
+    @objc
+    private func themeDidChange() {
+        self.update(theme: ThemeService.shared().theme)
+    }
+    
+    //  MARK: Themable
+    
+    func update(theme: Theme) {
+        backgroundColor = theme.colors.quinaryContent
+        imageView.tintColor = theme.colors.tertiaryContent
+        messageLabel.textColor = theme.colors.primaryContent
```

Comment:
Same thought about using the fonts from the theme.

Reply:
It makes more sense to use it here, as this is an independent view. Thanks for the suggestion.